src/v1/controllers/TenantCreate/createTenantValidationSchema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "type": "object",
+  "title": "tenant",
+  "required": ["tenant_name", "tenant_type"],
+  "properties": {
+    "tenant_name": {
+      "type": "string",
+      "description": "must be a string and is required"
+    },
+    "tenant_type": {
+      "type": "string",
+      "description": "must be a string and is required"
+    },
+    "is_active": {
+      "type": "boolean",
+      "description": "must be a boolean and is required"
+    },
+    "status": {
+      "type": "string",
+      "enum": ["live", "draft"],
+      "description": "must be a string and one of the predefined values"
+    },
+    "created_by": {
```

Comment:
created at, updated at?

Reply:
created_at and updated_at are created automatically when the record is inserted.